arangod/RestHandler/RestMetricsHandler.cpp

```diff
@@ -148,23 +131,32 @@ RestStatus RestMetricsHandler::execute() {
 
   auto& metrics = server().getFeature<metrics::MetricsFeature>();
   if (!metrics.exportAPI()) {
-    // dont export metrics, if so desired
+    // don't export metrics, if so desired
     generateError(rest::ResponseCode::NOT_FOUND, TRI_ERROR_HTTP_NOT_FOUND);
     return RestStatus::DONE;
   }
 
-  std::vector<std::string> const& suffixes = _request->suffixes();
-
-  bool v2 = false;
-  if (suffixes.size() > 0 && suffixes[0] == "v2") {
-    v2 = true;
+  auto const& suffixes = _request->suffixes();
+  auto const& values = _request->values();
+  bool const v2 = !suffixes.empty() && suffixes[0] == "v2";
+  if (values.contains("json")) {
+    TRI_ASSERT(v2);
+    bool const coordinator = values.contains("coordinator");
```

Comment:
This only checks if `json` / `coordinator` are used as URL parameters, but does not check their values.
I think we should here check their boolean values, so that one can call the API with `json=true` or `json=false`. We can use `basics::StringUtils::boolean(...)` for parsing the parameter value. That would be consistent with most other APIs then.

Reply:
Ok

Reply:
![image](https://user-images.githubusercontent.com/32071355/154489775-eebe583d-df9b-491a-b027-5ca9af69bc57.png)

```C++
  if (str.empty()) {
    return false;
  }
  std::string lower = trim(str);
  tolowerInPlace(lower);

  if (lower == "true" || lower == "yes" || lower == "on" || lower == "y" ||
      lower == "1" || lower == "✓") {
    return true;
  }
  return false;
```